runtime/environment-cpu.yml

```diff
@@ -38,3 +38,8 @@ dependencies:
       - git+https://github.com/ibarrond/Pyfhel.git@0dee7a77469627c4de4f2dca97cdc68bf006a77d
       - tensorflow_privacy==0.7.3
       - tensorflow-probability==0.15.0
+      - deepwalk==1.0.3
+      - imblearn==0.0
+      - gensim==3.8.3
```

Comment:
This version of gensim has been included as part of another PR that has been merged in, so you should be all set on this one. See #33.